bot/check-build.sh

```diff
@@ -417,141 +428,149 @@ success_msg="found message(s) matching <code>${GP_no_missing}</code>"
+        if [[ ! -z ${software_pkgs} ]]; then
+            while IFS= read -r sw_pkg ; do
+                comment_artifacts_list="${comment_artifacts_list}$(print_br_item '<code>__ITEM__</code>' ${sw_pkg})"
+            done <<< "${software_pkgs}"
+        else
+            comment_artifacts_list="${comment_artifacts_list}$(print_br_item '__ITEM__' 'no software packages in tarball')"
+        fi
+        comment_artifacts_list="${comment_artifacts_list}</pre>"
+        comment_artifacts_list="${comment_artifacts_list}$(print_br_item 'other under ___ITEM___' ${prefix})"
+        comment_artifacts_list="${comment_artifacts_list}<pre>"
+        if [[ ! -z ${other_shortened} ]]; then
+            while IFS= read -r other ; do
+                comment_artifacts_list="${comment_artifacts_list}$(print_br_item '<code>__ITEM__</code>' ${other})"
+            done <<< "${other_shortened}"
+        else
+            comment_artifacts_list="${comment_artifacts_list}$(print_br_item '__ITEM__' 'no other files in tarball')"
+        fi
+        comment_artifacts_list="${comment_artifacts_list}</pre>"
     else
-        comment_artifacts_list="${comment_artifacts_list}$(print_br_item '__ITEM__' 'no other files in tarball')"
+        comment_artifacts_list="${comment_artifacts_list}$(print_dd_item 'No artefacts were created or found.' '')"
     fi
-    comment_artifacts_list="${comment_artifacts_list}</pre>"
-else
-    comment_artifacts_list="${comment_artifacts_list}$(print_dd_item 'No artefacts were created or found.' '')"
-fi
 
-comment_artefact_details_fmt="<details>__ARTEFACT_SUMMARY____ARTEFACT_DETAILS__</details>"
-comment_artefacts_details="${comment_artefact_details_fmt/__ARTEFACT_SUMMARY__/${artefact_summary}}"
-comment_artefacts_details="${comment_artefacts_details/__ARTEFACT_DETAILS__/${comment_artifacts_list}}"
+    comment_artefact_details_fmt="<details>__ARTEFACT_SUMMARY____ARTEFACT_DETAILS__</details>"
+    comment_artefacts_details="${comment_artefact_details_fmt/__ARTEFACT_SUMMARY__/${artefact_summary}}"
+    comment_artefacts_details="${comment_artefacts_details/__ARTEFACT_DETAILS__/${comment_artifacts_list}}"
+
+    comment_artefacts_fmt="<dt>_Artefacts_</dt><dd>__ARTEFACTS_LIST__</dd>"
+    comment_artefacts="${comment_artefacts_fmt/__ARTEFACTS_LIST__/${comment_artefacts_details}}"
+    comment_description=${comment_description/__ARTEFACTS_FMT__/${comment_artefacts}}
 
-comment_artefacts_fmt="<dt>_Artefacts_</dt><dd>__ARTEFACTS_LIST__</dd>"
-comment_artefacts="${comment_artefacts_fmt/__ARTEFACTS_LIST__/${comment_artefacts_details}}"
-comment_description=${comment_description/__ARTEFACTS_FMT__/${comment_artefacts}}
+    echo "artefacts = " >> ${job_result_file}
+    echo "${TARBALL}" | sed -e 's/^/    /g' >> ${job_result_file}
+
+    # remove tmpfile
+    if [[ -f ${tmpfile} ]]; then
+        rm ${tmpfile}
+    fi
+
+elif [[ -f "$TOPDIR/check-build-artefacts.sh" ]]; then
+    source "$TOPDIR/check-build-artefacts.sh"
+fi
```

Comment:
@smoors There should be an else here, in case `--use-check-build-artefacts-script` is used but not corresponding script is available:
```suggestion
else
    echo "ERROR: Required script $TOPDIR/check-build-artefacts.sh not found!" >&2
    exit 1
fi